src/iso_alloc_interfaces.c

```diff
@@ -201,6 +201,16 @@ EXTERNAL_API NO_DISCARD void *iso_alloc_untag_ptr(void *p, iso_alloc_zone_handle
     return _untag_ptr(p, zone);
 }
 
+EXTERNAL_API void iso_alloc_verify_ptr_tag(void *p, iso_alloc_zone_handle *zone) {
```

Comment:
Shouldn't this be a `return` if `MEMORY_TAGGING` isn't set?

Reply:
I keep all the implementation details, include compile time excludes and likely/unlikely macros, to the internal code to keep the interfaces file from getting too complex. It makes it easier for me to reason about the API contracts.